src/components/HdDashedList.vue

```diff
@@ -69,6 +68,7 @@ export default {
 
 .dashed-list {
   $root: &;
+  $leaders-border: 2px dashed #D6DDE6;
```

Comment:
```suggestion
  $leaders-border: 2px dashed getShade($neutral-gray, 50);
```

I think we are able to use the design system colors here.
https://zeroheight.com/058baf035/p/120a4f-colors/t/44ef55

Then we can achieve the shade values by calling this function:
https://github.com/homeday-de/homeday-blocks/blob/develop/src/styles/_colors.scss#L2

But I might be wrong 😅